apps/site/assets/ts/models/__tests__/alert-test.ts

```diff
@@ -44,6 +45,17 @@ export const beforeNow = () => {
   ];
 };
 
+export const threeDaysFromNow = () => {
+  const now = Date.now();
+  const five_minutes = 300_000;
+  return [
+    [
+      activePeriodDateFormatted(now - five_minutes),
```

Comment:
This can be simplified as `activePeriodDateFormatted(add(now, { minutes: -5}).getMilliseconds())`